core/src/main/scala/kafka/tools/StreamsResetter.java

```
@@ -136,6 +150,22 @@ public int run(final String[] args, final Properties config) {
         return exitCode;
     }
 
+    private Properties parseKeyValues(final List<String> consumerProperties) {
+
+        final Properties parsedConsumerProperties = new Properties();
+        for (final String val: consumerProperties) {
+            final String[] property = val.split("=");
+            if (property.length == 2) {
+                parsedConsumerProperties.put(property[0], property[1]);
+            } else {
+                System.err.println("Invalid command line properties: " + consumerProperties.toString());
+                System.exit(1);
```

We should just throw an "InvalidArgumentException" here.